HashSet.py

```diff
@@ -0,0 +1,64 @@
+# Time Complexity : O(1) for searching and adding
+# Space Complexity : O(1)
+# Did this code successfully run on Leetcode : Yes
+# Any problem you faced while coding this : No
+
+#Used the logic of HashSets in Java and implemented tehir storage using two classes for Buckets and the actual Hashset.
+
+class Bucket:
+    def __init__(self):
```

Comment:
This will be O(n) solution because u have to search for it, instead make each bucket some fixed size array and based on the second hash add it in the respective space